src/effects/DtmfGen.h

```diff
@@ -94,7 +111,6 @@ class EffectDtmf final : public Effect
    struct Validator;
 
```

Comment:
Github will not allow me to comment on line 102 so I need to do it here: 
On msvc2019 I get this error:

`error C2397: conversion from 'size_t' to 'int' requires a narrowing conversion`